tests/conftest.py

```diff
@@ -1,11 +1,22 @@
+from importlib import import_module
+
 import django
 import pytest
 from django.core import management
 
 
-def pytest_configure():
+def pytest_configure(config):
     from django.conf import settings
 
+    config.addinivalue_line(
+        "markers", "contrib(name): mark required contrib package"
+    )
+
+    contrib_apps = [
+        'rest_framework_jwt',
```

Comment:
and rest_framework_simplejwt ?

not sure I understand why include some and not others.

Reply:
that is a good question. afaik some django "apps"/plugins don't care and others are even not supposed to be listed there. i think i listed the ones that explicitly required it. in case of ``rest_framework_simplejwt``, the documentation gives no indication that it should be added. ``polymorphic`` on the other hand does and that is likely an oversight on my part.

Reply:
I would add all, unless there is a problem with one.

And if there is a problem, be suspect of the app.

Typically the reason why they dont want to be listed, is that their top-level `__init__` imports "deep" Django/DRF stuff (and not lazily) and import fails if listed in INSTALLED_APPS, because those imports should have been done at `AppConfig.ready()`, or done lazily.  They are relying on the invoking app to only import their package at the appropriate phase in Django startup, shifting the burden onto the caller.

This means they can instruct users to do `from my_funky_app import ThisNewField` when they should have put the field into a separate module and the user would need to do `from my_funky_app.fields import ThisNewField`.  I've seen it a bit in DRF 'helpers' that dont have any models or register-able classes.  As soon as this helper gets more complicated (gets better tests/support more versions of Django/DRF), they need to do a compatibility-breaking cleanup to empty out `__init__` so they can get onto the INSTALLED_APPS startup wagon.

Reply:
that is all good and well but i don't want to create problem where there aren't any. if the app documentions says so, i will add the app. added ``polymorphic``